src/components/Timeline/updates/TimelineJourneyMilestone.tsx

```diff
@@ -0,0 +1,127 @@
+import CancelIcon from '@material-ui/icons/Cancel';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import FlagIcon from '@material-ui/icons/Flag';
+import { FormattedMessage } from 'react-intl';
+import React from 'react';
+import { Box, Grid, Typography } from '@material-ui/core';
+
+import PersonHoverCard from 'components/PersonHoverCard';
+import theme from 'theme';
+import TimelineActor from '../TimelineActor';
+import { ZetkinPerson as ZetkinPersonType } from 'types/zetkin';
+import ZetkinRelativeTime from 'components/ZetkinRelativeTime';
+import {
+  CHANGE_PROPS,
+  UPDATE_TYPES,
+  ZetkinUpdateJourneyMilestone,
+} from 'types/updates';
+
+const getPersonName = (person: Partial<ZetkinPersonType>) =>
+  person.first_name + ' ' + person.last_name;
+
+interface Props {
+  update: ZetkinUpdateJourneyMilestone;
+}
+
+const TimelineJourneyMilestone: React.FunctionComponent<Props> = ({
+  update,
+}) => {
+  let changeToRender: 'complete' | 'incomplete' | 'deadline';
```

Comment:
Could this union type go in `types/updates.ts`?

Reply:
Overkill imo